common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Strategy.java

```diff
@@ -615,6 +618,12 @@ protected TokenResult getTokenResultFromHttpResponse(@NonNull final HttpResponse
                     tokenResponse.setCliTelemSubErrorCode(cliTelemInfo.getServerSubErrorCode());
                 }
             }
+
+            if (null != responseHeaders.get(XMS_CCS_REQUEST_ID)) {
+                Span.current().setAttribute(
```

Comment:
Do we also want to emit this in JoinedAccountRequestHandler? I believe that code path doesn't use the strategies...can you double check?